ls1intum/Artemis
# Feature/#1369 add rating to text exercise

### Checklist

### Motivation and Context
This feature allows students to rate the feedback they got on a result level in text exercises.

### Description
- Created a RatingModule that can be used for every exercise 
- Added a REST route
- Inserted the newly created RatingComponent into the text exercises and styled it

### Steps for Testing


1. Log in to Artemis
2. Navigate to a text-exercise that has feedback
2.1 If such does not exist do the following:
2.2 Create a new Text exercise
2.3 Submit a solution for it
2.4 Assess this text exercise
3. Navigate to "view result" on that text exercise
4. See 5 empty stars at the bottom of the page
5. Rate the result
6. Update the rating by giving it another amount of stars
6. Reload the page and see that the rating was saved

### Screenshots


<img width="1440" alt="image" src="https://user-images.githubusercontent.com/51994246/82139073-a5bc1480-9825-11ea-9a41-7eb902f2c5d3.png">
